tests/test_serialization.py

```diff
@@ -48,7 +48,8 @@ def validate_page(self, fp):
 
     ids = [x.get('id') for x in doc.findall('.//*[@id]')]
     counts = Counter(ids)
-    self.assertEqual(counts.most_common(1)[0][1], 1, msg='Duplicate IDs in PageXML output')
+    if len(counts):
```

Comment:
Had to add this check as `doc.findall('.//*[@id]')` is not returning any element in the segmentation test